src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_RefReadOnly.cs

```diff
@@ -29,7 +32,19 @@ class Test
 }
 ";
 
-            CompileAndVerify(text, verify: Verification.Fails, symbolValidator: module =>
+            void validatePE(PEAssembly assembly)
+            {
+                var reader = assembly.GetMetadataReader();
+                var test = reader.GetTypeDefinitionHandleByName("Test");
+                var method = reader.GetMethodDefinitionByName(reader.GetTypeDefinition(test), "M");
+                var returnTypeHandle = method.GetParameters().ElementAt(0);
```

Comment:
>method.GetParameters().ElementAt(0) [](start = 39, length = 35)

Symbol for return parameter is available through PEMethodSymbol.Signature.ReturnParam

Reply:
Thanks. That is indeed much nicer. I assume it's ok for me to make `Signature` and its return type `SignatureData` internal for testing purpose.

---
In reply to: [168326601](https://github.com/dotnet/roslyn/pull/24492#discussion_r168326601) [](ancestors = 168326601)